ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -497,7 +497,12 @@ def process_cluster_cli_params(config):
     suffix = ocsci_config.cur_index + 1 if ocsci_config.multicluster else ""
     cluster_path = get_cli_param(config, f"cluster_path{suffix}")
     if not cluster_path:
-        raise ClusterPathNotProvidedError()
+        if "KUBECONFIG" in os.environ:
+            cluster_path = os.path.dirname(os.environ["KUBECONFIG"])
```

Comment:
Do we need to check if `os.path.exists()`?

Reply:
```oviner:~$ printenv | grep -i kube
KUBECONFIG=/home/oviner/ClusterPath/auth/kubeconfig
```
We are checking the value of `KUBECONFIG`